assets/src/blocks/HubspotForm/hooks/useBackgroundImage.js

```diff
@@ -0,0 +1,11 @@
+import { useState, useEffect } from '@wordpress/element';
+
+export const useBackgroundImage = (image) => {
+  const [ backgroundImage, setBackgroundImage ] = useState();
+
+  useEffect(() => {
+    setBackgroundImage({ backgroundImage: image ? `url(${image})` : 'none' });
```

Comment:
You could replace this whole function with just this:
```
export const useBackgroundImage = (image) => {
  return { backgroundImage: image ? `url(${image})` : 'none' };
}
```
and it would work just as well, only the first time it wouldn't return `null` for the background.

String interpolation is a very cheap operation. Much cheaper than the overhead the `useState` and `useEffect` hooks add.

Using an effect would only make sense here if you called an API to get the specific background each time, or do some heavy calculation.